user/src/api/fireEquipmentApi.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  useAuthenticatedGet,
+  useAuthenticatedPatch,
+  useAuthenticatedPost,
+} from '@/hooks/useApi';
+import { ApiResponse } from './api';
+
+export enum FireEquipmentFuel {
+  GAS = 1,
+  KEROSENE = 2,
+  CHARCOAL = 3,
+}
+
+export type FireEquipmentResponse = {
+  id?: number;
+  name: string;
+  quantity: number;
+  fuel: FireEquipmentFuel;
+  usage: string;
+  is_takeaway: boolean;
+  remark?: string;
+  group_id: number;
+  created_at?: string;
+  updated_at?: string;
+};
+
+const API_ENDPOINTS = {
+  FIRE_EQUIPMENT_PATH: '/fire_equipment',
+};
+
+// グループIDで火器使用申請を取得
+export const useGetFireEquipmentOrderByGroupId = (
+  groupId: number | undefined
+) => {
+  const endpoint =
+    groupId !== undefined
+      ? `${API_ENDPOINTS.FIRE_EQUIPMENT_PATH}/group/${groupId}`
```

Comment:
好み分かれそうだけど、揃えたほうが分かりやすいからこっち派
まぁどっちでもおｋ 🙆‍♂️ 

```suggestion
      ? `${API_ENDPOINTS.FIRE_EQUIPMENT_ORDERS}/group/${groupId}`
```

Reply:
[命名など変更](https://github.com/NUTFes/group-manager-2/pull/1918/commits/3f790a6283751123bc7bf27bbc4cd35e09cb3278)